charts/primary-site/Chart.yaml

```diff
@@ -13,6 +13,6 @@ type: application
 # 1.0.0-alpha.0
 # 1.0.0-alpha.1
 # 1.0.0
-version: "0.0.53"
+version: "0.0.54"
 
 appVersion: "b5ab5dc7bd5ef765d43830423ea4465d3bb016b1"
```

Comment:
This doesn't need to change?

Reply:
No, it doesn't this is a chart template only change from #106